### ISSUE TITLE
MS-SSIM Fine-tuning epochs

### ISSUE THREAD

**Issue by Indraa145** [InterDigitalInc/CompressAI#261]
Hello, in the CompressAI documentation, it is mentioned that "MS-SSIM optimized networks were fine-tuned from pre-trained MSE networks (with a learning rate of 1e-5 for both optimizers)."

I want to ask how many training epochs did you do the fine-tuning for?

Thank you.

**Comment by Indraa145** [comment]
Also, while I finetune the model, it seems while the overall loss do get lower, my SSIM value only increase slightly, while my PSNR value keep going down as more iteration goes on.
